classes/controller/AdminController.php

```diff
@@ -2701,7 +2701,10 @@ public function setMedia($isNewTheme = false)
             $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/' . $this->bo_theme . '/css/vendor/titatoggle-min.css', 'all', 0);
             $this->addCSS(__PS_BASE_URI__ . $this->admin_webpath . '/themes/' . $this->bo_theme . '/public/theme.css', 'all', 0);
 
-            $this->addJquery();
+            // add Jquery 3 and its migration script
+            $this->addJs(_PS_JS_DIR_ . 'jquery/jquery-3.4.1.min.js');
+            $this->addJs(_PS_JS_DIR_ . 'jquery/jquery-migrate-3.1.0.min.js');
```

Comment:
@eternoendless @PierreRambaud 

So I deprecated the addJquery method and called jquery 3 directly in AdminController, with other javascript assets.

I just wonder if I should still put the jquery version in a constant, like it used to be? It would make sense if for some reason we wanted back office and front office to alway share the same jquery version, I don't think it's the case, is it?

Reply:
Not sure it's useful since jquery migrate requires a jquery versions range :thinking: 
More, just asking, but the jquery and jquery.migrate dependencies shouldn't be in package.json?

Reply:
I just cheked and there is no mention of jquery in admin default theme's package.json.

Reply:
Maybe you can use npm to install these files instead of hard copy them?

Reply:
If we use package.json without compilation, this means we have the node_modules directory. So we can't add this dependencies right now :/